pyro/infer/trace_elbo.py

```diff
@@ -166,18 +165,11 @@ class JitTrace_ELBO(Trace_ELBO):
     """
     def loss_and_grads(self, model, guide, *args, **kwargs):
         if getattr(self, '_loss_and_surrogate_loss', None) is None:
-            # populate param store
-            with poutine.block():
-                with poutine.trace(param_only=True) as param_capture:
-                    for _ in self._get_traces(model, guide, *args, **kwargs):
-                        pass
-            self._param_names = list(param_capture.trace.nodes.keys())
-
             # build a closure for loss_and_surrogate_loss
             weakself = weakref.ref(self)
 
-            @torch.jit.compile(nderivs=1)
-            def loss_and_surrogate_loss(args_list, param_list):
+            @pyro.ops.jit.compile(nderivs=1)
+            def loss_and_surrogate_loss(*args_list):
```

Comment:
nit: I guess we can just call this `*args` now to match `Trace_ELBO.loss_and_grads()` idioms?